src/goto-checker/symex_coverage.cpp

```diff
@@ -14,6 +14,7 @@ Date: March 2016
 #include "symex_coverage.h"
 
 #include <chrono>
+#include <ctime>
```

Comment:
Thank you for fixing this! Would you mind moving it to a PR of its own, I'll happily add my approval as soon as I get to see the PR. And: my apologies for messing this up.

Reply:
Yeah, no problem - I will spin this out to its own PR now.

Also, yeah it's hard to notice subtle breakages like this - I only discovered it yesterday when I played around with VS on windows to write these docs.